lua/lvim/core/alpha.lua

```diff
@@ -23,6 +33,10 @@ local function resolve_buttons(theme_name, entries)
     local button_element = selected_theme.button(entry[1], entry[2], entry[3])
     -- this became necessary after recent changes in alpha.nvim (06ade3a20ca9e79a7038b98d05a23d7b6c016174)
     button_element.on_press = on_press
+
+    button_element.opts =
+      vim.tbl_extend("force", button_element.opts, entry[4] or lvim.builtin.alpha[theme_name].default_button_opts)
```

Comment:
I think it's fine for now, since `buttons.opts` only has `spacing`, we could probably upstream this

```suggestion
      vim.tbl_extend("force", button_element.opts, entry[4] or lvim.builtin.alpha[theme_name].section.buttons.opts)
```